workspaces/orchestrator/plugins/orchestrator-form-widgets/src/FormWidgetsApi.tsx

```diff
@@ -68,4 +68,12 @@ export class FormWidgetsApi implements OrchestratorFormApi {
       };
     };
   };
+
```

Comment:
The orchestrator-form-widgets is not just an example of how to use OrchestratorFormAPi, it's actually used as is by customers and provides many capabilities. 
Perhaps this is not clear from the documentation.
putting getReviewComponent here, will change the behavior for these customers.
To showcase this capability another example plugin is required unfortunately.